src/components/projects/CreateProjectForm.vue

```diff
@@ -362,34 +363,54 @@
               style="width: 150px;"
             >
               <b-numberinput
+                v-if="exposures[n-1].zoom === 'Mosaic deg.' || exposures[n-1].zoom === 'Mosaic arcmin.'"
                 :value="Number(exposures[n-1].width)"
                 :class="getSymbol(exposures[n-1].zoom)"
                 size="is-small"
                 :disabled="!exposures[n-1].active"
                 type="number"
-                :min="exposures[n-1].zoom === 'Mosaic arcmin.' ? minWidthDegrees * degreesToArcminutes : minWidthDegrees"
-                :max="exposures[n-1].zoom === 'Mosaic arcmin.' ? maxWidthDegrees * degreesToArcminutes : maxWidthDegrees"
+                :min="minDegrees"
+                :max="exposures[n-1].zoom === 'Mosaic arcmin.' ? maxDegrees * degreesToArcminutes : maxDegrees"
                 :step="exposures[n-1].zoom === 'Mosaic arcmin.' ? conditionalStep * 10 : conditionalStep"
                 min-step="0.001"
                 @input="val => exposures[n-1].width = val"
               />
+              <b-numberinput
+                v-else
+                :value="Number(exposures[n-1].width)"
+                :class="getSymbol(exposures[n-1].zoom)"
+                size="is-small"
+                :disabled="true"
+                type="number"
+                @input="val => exposures[n-1].width = val"
```

Comment:
I think for the input handler, we could instead use

```
v-model="exposures[n-1].width"
```

which _should_ map exposures[n-1].width to the value of number input? 

https://vuejs.org/guide/components/v-model.html#component-v-model